samples/tab-mail/ts/README.md

```diff
@@ -0,0 +1,93 @@
+---
+page_type: sample
+description: Sample tab-mail app.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+extensions:
+ contentType: samples
+ createdDate: "20/06/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-mail-ts
+---
+
+# Tab-Mail
+
+This sample helps to compose mail from perosnal tab.
```

Comment:
This sample app shows how to compose mail in Outlook using personal tab app.

Reply:
fixed